### DISCUSSION TITLE
Versioning of storage

### DISCUSSION THREAD

**Discussion by aharwood9** [codemask-labs/stan-js#82] [Q&A] [answered]
Hi there 👋 Thank you for all the great work on the library. I am just wondering if there is a way to version the storage when you introduce breaking changes. Similar to this which I am currently using - https://zustand.docs.pmnd.rs/integrations/persisting-store-data#version

**Comment by ceski23** [comment, marked as answer]
Hi, thanks for trying out stan-js 😄 for now there is nothing built-in like that because we want to keep library lean, but we tried to make API flexible enough to allow for such extensions. [Here](https://www.npmjs.com/package/@ceski23/stan-js-utils) is package with some utils for stan-js that contains `withMigrations` util which will allow you to version storage and safely migrate data
